source/tests/universal/common/cases/loss/utils.py

```
@@ -36,14 +41,20 @@ def test_forward(self):
         nframes = 2
 
         def fake_model():
-            model_predict = {
-                data_key: fake_input(
-                    label_dict[data_key], natoms=natoms, nframes=nframes
-                )
-                for data_key in label_keys
-            }
-            if "atom_ener" in model_predict:
-                model_predict["atom_energy"] = model_predict.pop("atom_ener")
+            if "property_name" in self.input_dict.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)